src/TextField/TextField.js

```diff
@@ -47,6 +52,14 @@ export default class TextField extends Component {
      * Whether the label should be displayed as required (asterisk).
      */
     required: PropTypes.bool,
+    /**
+     * Input type.
```

Comment:
This feels  a bit terse. (I know it's an HTML property, and we don't normally explicitly expose or document those, but it may not be immediately obvious what the supported values are.)